pacman/README.md

```diff
@@ -0,0 +1,155 @@
+# Credits
+
+Credits: [UC Berkeley](http://ai.berkeley.edu/project_overview.html)
+         [OpenAI Interface](https://github.com/sohamghosh121/PacmanGym)
```

Comment:
Put credits at the end. 

Instead make the title as `# Pacman project`. Then below add a table of content.

Reply:
Also explain the project is divided into three parts.